addons/hr_expense/tests/common.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.addons.account.tests.test_invoice_tax_totals import TestTaxTotals
 from odoo.addons.mail.tests.common import mail_new_test_user
 
 
-class TestExpenseCommon(AccountTestInvoicingCommon):
+class TestExpenseCommon(TestTaxTotals):
```

Comment:
Don't change that. You are testing twice the same unit tests in `TestTaxTotals`

Reply:
oh yeah, prior I changed it as I wanted to test totals widget. Forgot to change back